docs/pipeline-api.md

```diff
@@ -8658,11 +8714,7 @@ The names of the <code>params</code> must match the names of the <code>params</c
 <h3 id="tekton.dev/v1beta1.Param">Param
 </h3>
 <p>
-<<<<<<< HEAD
```

Comment:
I am surprised to see this kind of github tags here? 🤔

Reply:
its coming from https://github.com/tektoncd/pipeline/blob/e8f26371d22a3420458d23c2684b12a961df9198/docs/pipeline-api.md#param-1

Reply:
https://github.com/tektoncd/pipeline/issues/6210
I think we should prioritize this issue-- I will hopefully have some time to look at it